Towny/src/main/java/com/palmergames/bukkit/util/ItemLists.java

```diff
@@ -278,11 +282,49 @@ public Collection<String> getMaterialNameCollection() {
 			.includeList(PLANTABLES)
 			.add("TALL_GRASS","LARGE_FERN","VINE","TWISTING_VINES_PLANT","WEEPING_VINES_PLANT","NETHER_WART_BLOCK","CRIMSON_ROOTS","WARPED_ROOTS","NETHER_SPROUTS","BIG_DRIPLEAF","SMALL_DRIPLEAF").build();
 
+	/**
+	 * Minecraft uses 3 types of air
+	 */
+	public static final ItemLists AIR_TYPES = newBuilder()
+		.add("AIR")
+		.add("CAVE_AIR")
+		.add("VOID_AIR").build();
+
+	/**
+	 * List of all the banners
+	 */
+	public static final ItemLists BANNERS = newBuilder().endsWith("_BANNER").build();
+
+	/**
+	 * List of wall coral fans, players can fall through these
+	 */
+	public static final ItemLists CORAL_FANS = newBuilder()
+		.endsWith("_WALL_FAN")
+		.build();
+		
+	/**
+	 * List of solid blocks
+	 */
+	public static final ItemLists NOT_SOLID_BLOCKS = newBuilder()
+			.add("TRIPWIRE", "TRIPWIRE_HOOK")
+			.add("REDSTONE_WIRE","COMPARATOR","REPEATER","LEVER")
+			.includeList(CORAL_FANS)
+			.includeList(AIR_TYPES)
+			.includeList(SIGNS)
+			.includeList(BUTTONS)
+			.includeList(PRESSURE_PLATES)
+			.includeList(FENCE_GATES) //if open they are basically air
+			.includeList(TRAPDOORS) //if open they are basically air
+			.includeList(DOORS) ////if open they are basically air
+			.includeList(TORCHES)
+			.includeList(BANNERS)
+			.includeList(PLANTS).build();
+	
 	public static final ItemLists FALLING_BLOCKS = newBuilder()
 			.add("SAND", "RED_SAND", "GRAVEL", "SUSPICIOUS_SAND", "SUSPICIOUS_GRAVEL")
 			.endsWith("_CONCRETE_POWDER")
 			.build();
-
+	

```

Comment:
Unneeded diff.

Reply:
weirdly enough can't find it, probably the IDE fixing some indentation

Reply:
In this case your IDE is adding random indentation, should still be fixed